katsdpsigproc/tune.py

```diff
@@ -59,6 +59,8 @@
 _ScoreFunc = Callable[[int], float]
 _T = TypeVar("_T")
 
+KATSDPSIGPROC_TUNE_FALLBACK = os.getenv("KATSDPSIGPROC_TUNE_FALLBACK", "tune")
```

Comment:
Add a check that it is one of the known valid values, and either crash out, or issue a warning (and replace it with the default). I'm undecided which so use your judgement.